source/distributed/tria.cc

```diff
@@ -2677,21 +2670,16 @@ namespace parallel
 
           // actually do the refinement but prevent the refinement hook below
           // from taking over
```

Comment:
can you replace this comment please? Something like:
"execute the refinement to change the local mesh by calling the base class refinement function directly"